charts/aws-ebs-csi-driver/templates/clusterrole-csi-node.yaml

```diff
@@ -11,4 +11,4 @@ rules:
     verbs: ["get", "patch"]
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
-    verbs: ["list"]
+    verbs: ["get", "list", "watch", "update"]
```

Comment:
Remove `"update"`, discussed offline and concluded it's not required.

Otherwise lgtm

Reply:
Fixed ✅